compiler/crates/relay-bin/src/main.rs

```diff
@@ -56,6 +57,11 @@ struct CompileCommand {
     #[clap(long, short)]
     watch: bool,
 
+    /// A list of projects to compile.
+    /// If excluded, all projects will be compiled
+    #[clap(long)]
+    projects: Vec<String>,
```

Comment:
I would expect this to just be `project` and that you would specify it multiple times to specify multiple projects. Can you confirm what it actually looks like to pass multiple projects? I think it would be not great to have to pass multiple projects using one flag (how are you supposed to delineate them? commas? spaces?)

Reply:
yeah I think it doesn't do any space / command delimiting by default. I updated the language a bit to better explain.

Reply:
Cool. Can just make this `--project`?
```suggestion
    project: Vec<String>,
```

Reply:
@captbaritone Incase it isn't clear from the code. I kept the rust name as `projects` since it truly is a list of projects to compile. The clap name is `project` since the user can pass projects like so `--project projectA --project projectB`